lib/date_range_form_field.dart

```diff
@@ -88,7 +83,8 @@ class DateRangeField extends FormField<DateTimeRange> {
                   onChanged?.call(picked);
                 }
               }
-
+              // This will display hintText if provided and if state.value is null
```

Comment:
It's a bit misleading to put this comment before L87 instead of before L98. I'd change

Reply:
This should be fixed now